### PR TITLE
component lock (#590)

### PR DESCRIPTION
**Pull request by doerfli**

### CODE REVIEW

File: contracts/instance/Instance.sol [hunk 1]
```diff
@@ -186,12 +185,11 @@ contract Instance is
...
     }
 
-    function setTargetLocked(address target, bool locked)
+    function setLocked(address target, bool locked)
         external 
         onlyOwner()
     {
-        // TODO refactor
-        // _instanceAdmin.setTargetLockedByInstance(target, locked);
+        _componentService.setLockedFromInstance(target, locked);
```
Comment (rapidddenis) [inline contracts/instance/Instance.sol:192]: What are the reasons to follow Instance -> ComponentService -> InstanceAdmin? 
Proposal: call instance admin directly: Instance -> InstanceAdmin.lockComponentFromIntance()

Comment (doerfli) [reply contracts/instance/Instance.sol:192]: to keep it in line with the concept of only changing state through services.

Comment (rapidddenis) [reply contracts/instance/Instance.sol:192]: Would argue against because in this case lock by Instance depends from service which is upgradeable.

Comment (doerfli) [reply contracts/instance/Instance.sol:192]: so far we always changed state through service. don't see why we shouldn't do this here. Services are always upgradable.

**Pull request merged**